server.js

```diff
@@ -1,32 +1,52 @@
-import express from 'express'
+import express, { response } from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
+import goldenGlobesData from './data/golden-globes.json'
+
 
 // Defines the port the app will run on. Defaults to 8080, but can be 
 // overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
 const port = process.env.PORT || 8080
 const app = express()
 
-// Add middlewares to enable cors and json body parsing
+
+// Add middlewares to ensable cors and json body parsing
 app.use(cors())
 app.use(bodyParser.json())
 
 // Start defining your routes here
-app.get('/', (req, res) => {
-  res.send('Hello world')
+app.get('/', (request, response) => {
+  response.send('Welcome to Destinys Golden Globe-API! Here is the endpoint for all the nominations https://goldenglobeawards.herokuapp.com/nominations, here is for getting a specific year and which won that year https://goldenglobeawards.herokuapp.com/nominations/years/2010?won=false etc')
 })
 
+// Get all the nominations and categories
+app.get('/nominations', (request, response) => {
+  response.json(goldenGlobesData)
+})
+
+app.get('/nominations/years/:year', (request, response) => {
+  const year = request.params.year
+  const showWon = request.query.won
+  let nominationsFromYear = goldenGlobesData.filter((item) => item.year_award === +year)
+
+  if (showWon) {
+    nominationsFromYear = nominationsFromYear.filter((item) => item.win)
+  }
+  response.json(nominationsFromYear)
+})
```

Comment:
Endpoints should be named after what they return and not after how they are filtered or sorted. This could therefore be a query param under the nominations endpoint:
`nominations?year=2010&win=true`

Reply:
A took away nominations and took year/:year instead :)

Reply:
It's returning nominations, so a more suiting name would be `/nominations`. My suggestion was to have the year as a query param under your main endpoint (instead of having it as another endpoint :) )